client/state/data-layer/wpcom/sites/simple-payments/index.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Internal dependencies
+ */
+import {
+	SIMPLE_PAYMENTS_PRODUCTS_LIST,
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_ADD,
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_EDIT,
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_DELETE,
+} from 'state/action-types';
+import {
+	receiveProductsList,
+	requestingProductList,
+	successProductListRequest,
+	failProductListRequest,
+	receiveUpdateProduct,
+	receiveDeleteProduct,
+} from 'state/simple-payments/product-list/actions';
+import { isRequestingSimplePaymentsProductList } from 'state/selectors';
+import { metaKeyToSchemaKeyMap, metadataSchema } from 'state/simple-payments/product-list/schema';
+import wpcom from 'lib/wp';
+import debug from 'debug';
+
+/**
+ * Module variables
+ */
+const log = debug( 'calypso:middleware-simple-payments' );
+
+/**
+ * Reduce function for product attributes stored in post metadata
+ * @param { Object }  sanitizedProductAttributes object with all sanitized attributes
+ * @param { Object } current  item from post_meta to process
+ * @returns { Object } sanitizedProductAttributes
+ */
+function reduceMetadata( sanitizedProductAttributes, current ) {
+	if ( metaKeyToSchemaKeyMap[ current.key ] ) {
+		sanitizedProductAttributes[ metaKeyToSchemaKeyMap[ current.key ] ] = current.value;
+	}
+	return sanitizedProductAttributes;
+}
+
+/**
+ * Formats /posts endpoint response into a product list
+ * @param { Object } product raw /post endpoint response to format
+ * @returns { Object } sanitized and formatted product
+ */
+function customPostToProduct( product ) {
+	return Object.assign(
+		{
+			ID: product.ID,
+			content: product.content,
```

Comment:
Should this be `description: product.content`? 
Looks like the product schema is using `description`